### DISCUSSION TITLE
Spellcheck

### DISCUSSION THREAD

**Discussion by dweremeichik** [requarks/wiki#3568] [Error / Bug Report]
When editing a page, misspelled words should be highlighted offering corrections as is the default behaviour for a browser.

**To Reproduce**
1. Create or edit an existing page
2. Misspell a word

**Expected behavior**
The misspelled word should be highlighted, and offer suggestions for the correct spelling.

**Host Info:**
 - OS: [Docker]
 - Wiki.js version: [2.0.0-beta.275]
 - Database engine: [postgres 9.6.15]

**Client Info:**
- OS: [Debian Stable]
- Browser: [Firefox ESR]

**Comment by Codelica** [comment]
It looks like Codemirror is used in the Wiki?  If so, this may work, although it hasn't been updated in a while. https://github.com/sparksuite/codemirror-spell-checker
